### ISSUE TITLE
Swipe-right on touchscreen laptops triggers Chromes previous-page function.

### ISSUE THREAD

**Issue by drcmda** [GoogleChrome/ui-element-samples#20]
![image](http://i.imgur.com/kBY9Ynq.jpg)

Is there any way to disable it?

**Comment by paullewis** [comment]
Iiiiiiiinteresting. Wonder if we need `evt.preventDefault()` to account for that.

**Comment by MarcusJT** [comment]
Yes, this navigation feature/gesture of Chrome interferes with these samples and many other web apps with similar functionality and it's VERY annoying as it can completely break the user experience because it's not the expected outcome when swiping on a sidebar, etc.

There's a Chrome flag that users can disable but really there needs to be a way for developers to disable it in code when there's a risk of a clash - if not then it should be disabled by default instead of enabled as it clearly hasn't been thought through.
